.gitignore

```diff
@@ -81,6 +81,8 @@ vendor/
 # JIRA plugin
 atlassian-ide-plugin.xml
 
+# VSCode
+.vscode/
```

Comment:
This is the generated portion of this file. Best bet is to regenerate it with [gitignore](https://www.gitignore.io/api/go,tags,intellij+iml,visualstudiocode) which adds visualstudiocode support, or alternatively add .vscode to the beginning of this file, before the autogenerated parts.